docs/protocol/life_cycle_of_a_service.md

```diff
@@ -24,43 +24,52 @@ This section details the states of a service minted in the Autonolas Protocol, a
 
 A service that [has just been minted](./mint_packages_nfts.md#mint-a-service) will be in _Pre-Registration_ (of agent instances) state.
 
-**Available actions:**
+Available actions:
 
-* The service owner can make amendments by pressing the _Update_ button.
-* The service owner can advance to the _Active Registration_ state by pressing the _Activate Registration_ button.
+* **Update.** The service owner can make amendments on [some parameters of the service](./mint_packages_nfts.md#mint-a-service).
+* **Activate registration.** The service owner can transit the service to the _Active Registration_ state.
 
 ### Active Registration
 
 The service is waiting for agent operators to register their agent instances.
 
-**Available actions:**
+Available actions:
 
-* Agent operators can register their agent instances in the service by specifying their addresses (starting with `0x...`) and pressing the _Register Agents_ button. When the last agent instance slot is filled, the service state will transit to the _Finished Registration_ state automatically.
-* The service owner can terminate the service by pressing the _Terminate_ button.
+* **Register agents.** Agent operators can register their agent instances in the service by specifying their addresses (starting with `0x...`). When the last agent instance slot is filled, the service will transit to the _Finished Registration_ state automatically.
+
+    !!! warning "Important"
+
+        When registering agent instance addresses for a service, some conditions have to be met:
+
+        * The operator address (that is, the address of the wallet used to submit the registration transaction) must be different from the agent instance address(es) being registered.
+        * The operator address must not be used as agent instance address in any other service.
+        * The agent instance address(es) being registered must not be registered in any other service.
```

Comment:
In this line we can hint that to re-use agent instance addresses, operators can terminate and unbound services that are no longer in use. After unbounding the instance address can be reused.

Reply:
Done